go.mod

```diff
@@ -1,36 +1,36 @@
 module github.com/aquasecurity/trivy
 
-go 1.19
+go 1.18
```

Comment:
why did we downgrade golang?

Reply:
it was set yo 1.19 by mistake , Teppei said it should be 1.18

Reply:
The diff on GitHub seems broken. The main branch is 1.18. There should be no diff. Merging the main branch into this branch may help.

Reply:
ok , will rebase again

Reply:
In Trivy, we will squash and merge PRs. You don't have to mind the merge commit. You can just merge main like `git merge main`.